packages/apps-v5/src/commands/releases/index.js

```diff
@@ -1,171 +0,0 @@
-'use strict'
-
-const cli = require('heroku-cli-util')
-const stripAnsi = require('strip-ansi')
-
-async function run(context, heroku) {
-  const statusHelper = require('../../status_helper')
-  const time = require('../../time')
-  const {truncate} = require('lodash')
-
-  let optimizationWidth = 0
-
-  let descriptionWithStatus = function (d, r) {
-    const width = () => process.stdout.columns > 80 ? process.stdout.columns : 80
-    const trunc = (s, l) => {
-      if (process.stdout.isTTY) {
-        return truncate(s, {length: width() - (optimizationWidth + l), omission: '…'})
-      }
-
-      return s
-    }
-
-    const status = statusHelper.description(r, runningRelease, runningSlug)
```

Comment:
`statusHelper.description` only used the first arg. See the implications in a comment below.